src/pybind/mgr/k8sevents/README.md

```diff
@@ -34,44 +38,39 @@ Total :   4
 or, focus on the ceph specific events(audit & healthcheck) that are being tracked by the k8sevents module.
 ```
 [root@rhcs4-3 kube]# ceph k8sevents ceph
-Last Seen             Type      Count  Message                                              Event Object Name
+Last Seen (UTC)       Type      Count  Message                                              Event Object Name
 2019/09/20 04:32:55   Normal        1  Client 'client.admin' issued: ceph osd pool delete   mgr.audit.osd_pool_delete_
 2019/09/20 04:13:23   Normal        2  Client 'mds.rhcs4-2' issued: ceph osd blacklist      mgr.audit.osd_blacklist_
 Total :   2
 ```
 
 ## Sending events from a standalone Ceph cluster to remote Kubernetes cluster
-To test interaction from a standalone ceph cluster to a kubernetes environment, you need to make changes on the kubernetes cluster **and** the mgr hosts.
+To test interaction from a standalone ceph cluster to a kubernetes environment, you need to make changes on the kubernetes cluster **and** on one of the mgr hosts.
 ### kubernetes (minikube)
 We need some basic RBAC in place to define a serviceaccount(and token) that we can use to push events into kubernetes. The `rbac_sample.yaml` file provides a quick means to create the required resources. Create them with `kubectl create -f rbac_sample.yaml`
   
 Once the resources are defined inside kubernetes, we need a couple of things copied over to the Ceph mgr's filesystem.
-### ceph mgr hosts
-First, create a directory (default is `/etc/ceph/kube`)  
-This directory should have the following files;  
-| filename | purpose |
-| -------- | ------- |  
-| ca.crt | root cert of the kubernetes API server |  
-| token | file containing the token from the ServiceAccount in kubernetes |  
-| config | config file that points to the API server, ca crt and token files and declares the namespace to send events to |  
+### ceph mgr host
+On one of you mgr hosts, we'll need the ca.crt of the kubernetes API, and the token associated with the service account we're using to access the kubernetes API.  
```

Comment:
As far as I can tell, there is no requirement do to this on a mgr host.

Reply:
Agree - I can reword